src/cosmic_ray/operators/remove_named_arguments.py

```diff
@@ -0,0 +1,68 @@
+"""Remove named arguments
+Remove in function call, named arguments
+
+Because in a lot of vase, named arguments represents options or fields.
+
+The lost of an option or a field must imply application failure.
+
+Examples:
+    - subprocess.Popen call
+    - ORM (like django models)
+"""
+import re
+from parso.python.tree import PythonNode, Operator as tree_Operator, Newline
+from cosmic_ray.operators.operator import Operator
+from cosmic_ray.operators.util import ObjTest
+
+
+class RemoveNamedArguments(Operator):
+    re_skip = re.compile(r'.*pragma:.*mutation-no-remove.*')
+
+    def mutation_positions(self, node):
+        if ObjTest(node).match(PythonNode, type='arglist'):
+            for child in node.children:
+                if ObjTest(child).match(PythonNode, type='argument'):
+                    if not self._do_skip_argument(child):
+                        yield child.start_pos, child.end_pos
+
+    @classmethod
+    def _do_skip_argument(cls, node):
```

Comment:
As discussed in the conversation, I'd rather handle this using a more general, interceptor-based approach. Generally speaking, operators should no be concerned with whether they're "disabled"; that decision is handled at a different place.